src/theory/bv/theory_bv_rewrite_rules_operator_elimination.h

```diff
@@ -99,15 +115,15 @@ inline Node RewriteRule<SgeEliminate>::apply(TNode node)
 template <>
 inline bool RewriteRule<SltEliminate>::applies(TNode node)
 {
-  return (node.getKind() == kind::BITVECTOR_SLT); 
+  return (node.getKind() == kind::BITVECTOR_SLT);
 }
 
 template <>
 inline Node RewriteRule<SltEliminate>::apply(TNode node)
 {
   Trace("bv-rewrite") << "RewriteRule<SltEliminate>(" << node << ")"
                       << std::endl;
-  NodeManager *nm = NodeManager::currentNM();
+  NodeManager* nm = NodeManager::currentNM();
```

Comment:
Can you exclude whitespace changes for the parts of this file that you did not modify?  Otherwise this makes it hard to track changes.

Reply:
reverted